src/app/routes/clientRoutes.js

```diff
@@ -99,7 +99,13 @@ module.exports = (app, keycloak) => {
     '/:slug/explore-course', '/:slug/explore-course/*', '/:slug/signup', '/signup', '/:slug/sign-in/*',
     '/sign-in/*', '/download/*', '/accountMerge/*','/:slug/accountMerge/*', '/:slug/download/*', '/certs/*', '/:slug/certs/*', '/recover/*', '/:slug/recover/*'], redirectTologgedInPage, indexPage(false))
 
-  app.all(['*/dial/:dialCode', '/dial/:dialCode'], (req, res) => res.redirect('/get/dial/' + req.params.dialCode + '?source=scan'))
+  app.all(['*/dial/:dialCode', '/dial/:dialCode'], (req, res) => {
+    if (_.get(req, 'query.channel')) {
+      res.redirect(`/${envHelper.sunbird_portal_slugForProminentFilter}/get/dial/${req.params.dialCode}?source=scan`);
```

Comment:
This is not required
envHelper.sunbird_portal_slugForProminentFilter
Use _.get(req, 'query.channel') instead of above